src/components/modal/signupmodal/SignupModal.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+import React from "react";
+
+import {
+  SignupModalCSS,
+  HeaderTextCSS,
+  BodyTextCSS,
+  InputCSS,
+  LogoCSS,
+  CloseCSS,
+  InputContainerCSS,
+  SubscribeButtonCSS,
+} from "./SignupModalCSS";
+import Logo from "../../../assets/logos/logo-transparent.svg";
+import ClosedIcon from "../../../assets/icons/functional/grey-close.svg";
+
+const SignupModalComponent = styled.div`
+  ${SignupModalCSS}
+`;
+
+const HeaderText = styled.h1`
+  ${HeaderTextCSS}
+`;
+
+const BodyText = styled.p`
+  ${BodyTextCSS}
+`;
+
+const LogoImage = styled.img`
+  ${LogoCSS}
+`;
+
+const CloseIcon = styled.img`
+  ${CloseCSS}
+`;
+
+const InputContainer = styled.div`
+  ${InputContainerCSS}
+`;
+
+const Input = styled.input`
+  ${InputCSS}
+`;
+
+const SubscribeButton = styled.button`
+  ${SubscribeButtonCSS}
+`;
+
+const SignupModalContainer = ({ clickEvent }) => {
+  return (
+    <SignupModalComponent>
+      <LogoImage src={Logo} />
+      <CloseIcon src={ClosedIcon} />
```

Comment:
Make sure to add an onClick on the close icon as well. Maybe name it onCloseClick?